packages/react-table/src/components/BaseTable/BaseTableHeaderRow.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
```

Comment:
Can't this and `BaseTableBodyRow` be replaced with `BaseTableRow`?

Reply:
I could, but BaseTableBodyRow has some additional props that are not applicable to rows in the header
```
/** Makes the row expandable and determines if it's expanded or not */
  isExpanded?: boolean;
  /** Whether the row is editable */
  isEditable?: boolean;
```

Reply:
What happens if you set these on a row in `thead`? If it's not undefined behavior I'm in favor of removing this component.

Reply:
OK i kept it to a single Tr component :)